setup.cfg

```diff
@@ -8,4 +8,4 @@ exclude = *migrations*,build*
 statistics = true
 application-import-names = groups
 import-order-style = google
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D203,D204
```

Comment:
[PEP 257:  A package may be documented in the module docstring of the `__init__.py` file in the package directory.](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring)